### PR TITLE
feat: add timescaledb support to active record models

### PR DESCRIPTION
**Pull request by santiagodoldan**

### CODE REVIEW

File: lib/timescaledb/rails/extensions/active_record/schema_statements.rb [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
+        # Returns an array of hypertable names defined in the database.
+        def hypertables
+          query_values('SELECT hypertable_name FROM timescaledb_information.hypertables')
+        end
+
+        # Checks to see if the hypertable exists on the database.
+        #
+        #   hypertable_exists?(:developers)
+        #
+        def hypertable_exists?(hypertable)
```
Comment (pmaite88) [inline lib/timescaledb/rails/extensions/active_record/schema_statements.rb:17]: you could also consider to rename this to just `hypertable?`

Comment (santiagodoldan) [reply lib/timescaledb/rails/extensions/active_record/schema_statements.rb:17]: I tried to follow rails naming convention here, check here -> https://github.com/rails/rails/blob/b8d9a363201717e493c3d765d1240f2e3052e4a3/activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb#L59

File: lib/timescaledb/rails/model.rb [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        # @return [String]
+        def hypertable_name
+          table_name.split('.').last
+        end
+
+        # Returns the schema where hypertable is stored.
+        #
+        # @return [String]
+        def hypertable_schema
+          if table_name.split('.').size > 1
```
Comment (pmaite88) [inline lib/timescaledb/rails/model.rb:25]: schemas may have dots in them as well, please check that

Comment (santiagodoldan) [reply lib/timescaledb/rails/model.rb:25]: good catch, just pushed a change to cover that new use case, thanks

**Review by pmaite88 — APPROVED** [review]

**Pull request merged**